internal/kafka/internal/presenters/cloudproviders.go

```diff
@@ -19,12 +19,11 @@ func PresentCloudProvider(cloudProvider *api.CloudProvider) public.CloudProvider
 func PresentCloudRegion(cloudRegion *api.CloudRegion) public.CloudRegion {
 	reference := PresentReference(cloudRegion.Id, cloudRegion)
 	return public.CloudRegion{
-		Id:                     reference.Id,
-		Kind:                   reference.Kind,
-		DisplayName:            cloudRegion.DisplayName,
-		Enabled:                cloudRegion.Enabled,
-		SupportedInstanceTypes: getSupportedInstanceTypes(cloudRegion.SupportedInstanceTypes),
```

Comment:
We need to keep this in for now. 

Marking the field as "deprecated" is to notify clients/users using this API that this field will be removed. The field shouldn't be removed until the deprecation period is over.